### DISCUSSION TITLE
How to fix mactex-no-gui formula for mac m1?

### DISCUSSION THREAD

**Discussion by fredguth** [Homebrew/discussions#2205] [Casks] [answered]
When you `brew install --cask mactex-no-gui`, the installation still needs a change in .zshrc (and it is not just start a new shell). 

In .zhrc you have to add:

```zsh
####   MACTEX-NO-GUI fix from https://qiita.com/sprout2000/items/3fb7ee3878b4ad451fc3
if [[ $(uname -m) == "arm64" ]]; then
  typeset -U path PATH # https://stackoverflow.com/questions/41074146/
  path=(
    /opt/homebrew/bin(N-/)
    /opt/homebrew/sbin(N-/)
    /usr/bin
    /usr/sbin
    /bin
    /sbin
    /Library/TeX/texbin
    /Library/Apple/usr/bin
  )
fi
```

I would like to alert other users of this condition.  How can I help?

**Comment by SMillerDev** [comment, marked as answer]
I think a better solution is to submit a patch to the mactex developers that resolves the issue in their path helper.

**Reply by fredguth** [reply]
Good idea.
